app/models/singly_linked_list.rb

```diff
@@ -0,0 +1,77 @@
+class SinglyLinkedList
+  include Enumerable
```

Comment:
Yeah, that looks more familiar.  Hey, I'm trying (and failing) to connect
with you via my Adium IM client.
Are you logged in to Google at the moment?

I'm thinking, let's merge your PRs into little-stuff.  Then, if you're
willing/interested, please tell me what you think about my slightly
different RSpec test for my linked list?

I'm going over to ezcater this afternoon with my linked list (they've
already seen the original version of this) and the background enroller and
its tests/features, and some scheduling stuff I did back at the DMC
(previous job).

What'cha up to?  Having single and double-linked lists seems like a very
good idea, along with all kinds of other CS kata.

On Tue, Feb 2, 2016 at 10:27 AM, zfletch <notifications@github.com> wrote:

> In app/models/singly_linked_list.rb
> <https://github.com/dgoldhirsch/little-stuff/pull/3#discussion_r51582881>:
>
> > @@ -0,0 +1,77 @@
> > +class SinglyLinkedList
> > +  include Enumerable
>
> The code that does this is written in C (I think it's this function
> <https://github.com/ruby/ruby/blob/6ee8ec70696260fb0ac2ab4bf13f40ff003a8995/vm_eval.c#L1147>),
> but you can get the same behavior with Ruby code that looks like:
>
> def first(max)
>   count = 0
>   acc = []
>
>   each do |n|
>     break if count == max
>     acc << n
>     count += 1
>   end
>
>   accend
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dgoldhirsch/little-stuff/pull/3/files#r51582881>.
>